openapi/servicenow/client.bal

```diff
@@ -0,0 +1,477 @@
+        return response;
+    }
+    # Retrieves a specified set of Customer Service Management (CSM) cases.
+    #
+    # + sysparmDisplayValue - Data retrieval operation for reference and choice fields. Based on this value, retrieves the display value and/or the actual value from the database.
+    # + sysparmFields - Comma-separated list of fields to return in the response.
+    # + sysparmLimit - Maximum number of records to return. Unusually large sysparm_limit values can impact system performance. For requests that exceed this number of records, use the sysparm_offset parameter to paginate record retrieval.
+    # + sysparmOffset - Starting record index for which to begin retrieving records. Use this value to paginate record retrieval. This functionality enables the retrieval of all records, regardless of the number of records, in small manageable chunks.
+    # + sysparmQuery - Encoded query used to filter the result set.
+    # + sysparmReferenceFields - Whether or not to retrieve fields from reference tables such as account, contact, consumer, and sn_app_cs_social_social_profile. By default, these fields are returned if the user has read access to them, but this adds performance overhead. When retrieving a large number of case records, this overhead can be eliminated by setting sysparm_reference_fields=false.
+    # + accept - Data format of the response body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function getCSMCases(string? sysparmDisplayValue = (), string? sysparmFields = (), decimal? sysparmLimit = (), decimal? sysparmOffset = (), string? sysparmQuery = (), boolean? sysparmReferenceFields = (), string? accept = ()) returns json|error {
+        string  path = string `/api/sn_customerservice/case`;
+        map<anydata> queryParam = {"sysparm_display_value": sysparmDisplayValue, "sysparm_fields": sysparmFields, "sysparm_limit": sysparmLimit, "sysparm_offset": sysparmOffset, "sysparm_query": sysparmQuery, "sysparm_reference_fields": sysparmReferenceFields};
+        path = path + check getPathForQueryParam(queryParam);
+        map<any> headerValues = {"Accept": accept};
+        map<string|string[]> accHeaders = getMapForHeaders(headerValues);
+        json response = check self.clientEp-> get(path, accHeaders, targetType = json);
+        return response;
+    }
+    # Creates a new Customer Service Management (CSM) case.
+    #
+    # + payload - Field name and the associated value for each parameter to define in the specified record in JSON format.
+    # + accept - Data format of the response body.
+    # + contentType - Data format of the request body.
+    # + return - If successful, returns a JSON object. Otherwise returns the relevant error.
+    remote isolated function insertCSMCase(json payload, string? accept = (), string? contentType = ()) returns json|error {
```

Comment:
The function name must be `createCSMCase`

Reply:
Fixed by https://github.com/kasthuriraajan/ballerinax-openapi-connectors/commit/7d722e1656dd4084768a6a6f0aab1753b97a5d98